### PR TITLE
remove push template code, set ACC extension version to 3.0.0

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by spoorthipujariadobe — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -134,3 +142,24 @@ jobs:
 
       - store_test_results:
           path: code/campaignclassic/build/outputs/androidTest-results
+          
+  build-test-app:
```
Comment (spoorthipujariadobe) [inline .circleci/config.yml:146]: We need to also remove the `Assemble App` step on line 91-93

**Comment by spoorthipujariadobe** [comment]
Let's also remove `registerExtension` if it exists in the Github API docs

**Comment by spoorthipujariadobe** [comment]
@rymorale can we also change the namespace for the `campaignclassic` extension? It needs to be `com.adobe.marketing.mobile.campaignclassic`

**Comment by PravinPK** [comment]
Can we also change the testapp to point to released Assurance version 
https://github.com/adobe/aepsdk-campaignclassic-android/blob/dev-v3.0.0/code/testapp/build.gradle.kts#L42

**Comment by PravinPK** [comment]
One last change in readMe file-> under Development

We can change
- Open the project To open and run the project, open the code/build.gradle file in Android Studio
to
- Open the project To open and run the project, open the **_code/build.gradle.kts_** file in Android Studio

**Review by PravinPK — APPROVED** [review]

**Pull request merged**